__device-tests__/blocks/block.js

```diff
@@ -0,0 +1,89 @@
+/** @flow
+ * @format */
+
+import wd from 'wd';
+
+export default class Block {
```

Comment:
Kinda a nit but, can we name this class with something more specific than `Block`? I'm afraid the "block" term has already been overloaded with the introduction of Gutenberg so, better to add more verbosity in this case.

While at it, not sure what this class is about by the way. Is it a grouping of helper functions? Can we add some inline code comments to make it more clear?

Reply:
> Kinda a nit but, can we name this class with something more specific than Block? I'm afraid the "block" term has already been overloaded with the introduction of Gutenberg so, better to add more verbosity in this case.

I was a bit weary of it too but it just seemed natural to with it at the time, I've renamed to BlockInteraction now though

> While at it, not sure what this class is about by the way. Is it a grouping of helper functions? Can we add some inline code comments to make it more clear?

It was just intended to be a base class with all the common code for all the Block interactions to be built out. It mainly came out an OOP kind of thinking, with the latest update I now include a utils.js in a helpers folder, do you think it might be better to include a file like of block-utils.js. I've also included more inline comments 👍🏽 let me know if you think it still needs a little more detail